app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command_line: str) -> None:
+    command, old_file, new_file = command_line.split(" ")
```

Comment:
"split" method, splits the string by spaces by default

Reply:
Ah, U r right.
Fixed